daemon/prune.go

```diff
@@ -259,20 +256,20 @@ func (daemon *Daemon) clusterNetworksPrune(pruneFilters filters.Args) (*types.Ne
 
 // NetworksPrune removes unused networks
 func (daemon *Daemon) NetworksPrune(pruneFilters filters.Args) (*types.NetworksPruneReport, error) {
-	rep := &types.NetworksPruneReport{}
-	clusterRep, err := daemon.clusterNetworksPrune(pruneFilters)
-	if err != nil {
-		logrus.Warnf("could not remove cluster networks: %v", err)
-	} else {
-		rep.NetworksDeleted = append(rep.NetworksDeleted, clusterRep.NetworksDeleted...)
+	if _, err := getUntilFromPruneFilters(pruneFilters); err != nil {
```

Comment:
Oh I see… `getUntilFromPruneFilters` is called multiple time. I find that weird though. Is there a reason to not get tha `until` value here and pass it to the underlying functions (`clusterNetworksPrune`, `localNetworkPrune`) ?